definitions/checks/yum_exclude.rb

```diff
@@ -0,0 +1,23 @@
+class Checks::YumExclude < ForemanMaintain::Check
+  metadata do
+    label :check_yum_exclude_list
+    description 'Check if yum exclude list is configured'
+    tags :pre_upgrade
+  end
+
+  def run
+    assert(exclude_set?, 'The /etc/yum.conf has exclude list configured as below,'\
+          "\n  #{yum_exclude}"\
+          "\nUnset this as it can cause yum update or upgrade failures !")
+  end
+
+  def exclude_set?
+    yum_exclude == 'exclude=' || yum_exclude == 'exclude ='
```

Comment:
This call the grep twice in many cases and misses e.g. '# exclude=kernel*'. Could we use regexp instead like
```ruby
yum_exclude =~ /^exclude\s*=\s*\S+/
```

Reply:
I changed this to regular expression, miss from my side about commented exclude line.